src/message.rs

```diff
@@ -159,6 +159,10 @@ pub enum Method {
     Setup,
     /// Set parameter
     SetParameter,
+    /// Announce
```

Comment:
Similar to `PlayNotify`, please document here that this is only for RTSP 1.0.

Reply:
It's changed and done within a commit.